src/main/java/fi/aalto/cs/apluscourses/utils/Resources.java

```diff
@@ -55,8 +55,14 @@ public Properties getProperties(@NotNull String resourceName) throws ResourceExc
     return props;
   }
 
+  /**
+   * Returns a resource as a stream.
+   * @param resourceName Name of the resource.
+   * @return An {@link InputStream} to the given resource.
+   * @throws ResourceException If the resource could not be found.
+   */
   @NotNull
-  private InputStream getStream(@NotNull String resourceName) throws ResourceException {
+  public InputStream getStream(@NotNull String resourceName) throws ResourceException {
```

Comment:
why this change occurs?

Reply:
Previously, **o1.json** was read using file system access. That caused some compatibility issues when developing on Windows, so, as a quick fix, I changed it to be read from resources. I reused this function which was already implemented. To be able to use it, I changed the method public.

The whole thing is of course a temporary solution and "for developing purposes only" because eventually the course config files are downloaded from the web. This method can then be changed back to private, if that feels appropriate.